twake/backend/node/src/services/documents/services/index.ts

```diff
@@ -538,23 +552,7 @@ export class DocumentsService {
       });
 
       return children.getEntities().every(child => {
-        if (
-          child.access_info.unauthorized_entities.find(entity =>
-            entityIdentityCheck(entity.id, context),
-          )
-        ) {
-          return false;
-        }
-
-        if (
-          !child.access_info.authorized_entities.find(entity =>
-            entityIdentityCheck(entity.id, context),
-          )
-        ) {
-          return false;
-        }
-
-        return true;
+        return this.checkAccess(child.id, child, level, context, token);
```

Comment:
This part is wrong, we do not prevent access to a folder because we don't have access to one of the children, we'll cover this case in the "get" method (already done) so here we should `return true`